nx-workspace/apps/web/src/components/atoms/core/Alert.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import MAlert, {
+  AlertProps as MAlertProps,
+  Color,
+} from "@material-ui/lab/Alert";
+
+export interface AlertProps extends MAlertProps {
+  Color?: Color;
```

Comment:
Just double checking that `Alert` doesn't have style props for setting color?

Reply:
it has a root level prop for `Color` which is used if provided - otherwise it is set by `Severity`
Agree, this ambiguity is not good though - I've now lifted the type out and exported it directly.